ofdb-core/src/usecases/send_update_reminders.rs

```diff
@@ -0,0 +1,267 @@
+    formatter: &F,
+    target_contact: TargetContact,
+    unchanged_since: Timestamp,
+    reminder_interval: Duration,
+) -> Result<()>
+where
+    R: PlaceRepo + SubscriptionRepo + ReminderRepo + UserRepo,
+    G: EmailGateway,
+    F: EmailReminderFormatter,
+{
+    let outdated_places = find_places_not_updated_since(repo, unchanged_since)?;
+    let unsent_reminders =
+        find_unsent_reminders(repo, outdated_places, target_contact, reminder_interval)?;
+    let unsent_emails = create_emails(formatter, unsent_reminders);
+    send_emails(repo, email_gateway, unsent_emails);
+    Ok(())
+}
+
+#[derive(Debug, Clone, Copy)]
+pub enum TargetContact {
+    Owner,
+    Scout,
+}
+
+fn find_places_not_updated_since<R>(
+    place_repo: &R,
+    unchanged_since: Timestamp,
+) -> Result<Vec<Place>>
+where
+    R: PlaceRepo,
+{
+    let places = place_repo
+        .find_places_not_updated_since(unchanged_since)?
+        .into_iter()
+        .map(|(place, _)| place)
+        .collect();
+    Ok(places)
+}
+
+fn find_unsent_reminders<R>(
+    repo: &R,
+    outdated_places: Vec<Place>,
+    target_contact: TargetContact,
+    reminder_interval: Duration,
+) -> Result<Vec<Reminder>>
+where
+    R: SubscriptionRepo + ReminderRepo + UserRepo,
+{
+    let mut reminders = vec![];
+    match target_contact {
+        TargetContact::Owner => {
+            for p in outdated_places {
+                if let Some(email) = p.contact_email() {
+                    if send_new_reminder(repo, &p.id, email, reminder_interval) {
+                        reminders.push(Reminder {
+                            recipients: vec![email.clone()],
+                            last_change: p.created.at,
+                            place: p,
+                        });
+                    }
+                }
+            }
+        }
+        TargetContact::Scout => {
+            let subscriptions = repo.all_bbox_subscriptions()?;
+            let users = repo.all_users()?;
+            for p in outdated_places {
+                let scouts = get_scouts_subscribed_to_place(&p, &users, &subscriptions);
+                let mut recipients = vec![];
+                for s in scouts {
+                    let email = EmailAddress::from(s.email.clone());
+                    if send_new_reminder(repo, &p.id, &email, reminder_interval) {
+                        recipients.push(email);
+                    }
+                }
+                if !recipients.is_empty() {
+                    reminders.push(Reminder {
+                        recipients,
+                        last_change: p.created.at,
+                        place: p,
+                    });
+                }
+            }
+        }
+    }
+    Ok(reminders)
+}
+
+fn send_new_reminder<R>(
+    repo: &R,
+    place_id: &Id,
+    email: &EmailAddress,
+    reminder_interval: Duration,
+) -> bool
+where
+    R: ReminderRepo,
+{
+    match repo.find_last_sent_reminder(place_id, email).unwrap() {
```

Comment:
`unwrap()` -> `ok()` if you really want to silently ignore any errors. Errors should at least be logged as warnings.